Month3/Project1/Project1/Circle.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include "ArrayVectors.h"
```

Comment:
You're forced to include the header file of your whole program since you declare the Shape class in there. This is really messy.